### PR TITLE
EVDOC01-28 Identificar NESES al Roster de CV

### PR DESCRIPTION
**Pull request by stetsche**
Amaga número d'expedient i pla d'estudis a la interfície d'usuari
Reordena les columnes a l'exportació
Canviar les traduccions

### CODE REVIEW

File: roster2/tool/src/java/org/sakaiproject/roster/tool/entityprovider/RosterPOIEntityProvider.java [hunk 1]
```diff
@@ -478,22 +478,22 @@ private void addOverviewRows(final List<List<String>> dataInRows,
...
+				row.add(additionalNotes != null ? String.join("\n", additionalNotes) : "");
 			}
 
 			if (this.sakaiProxy.getViewCandidateDetails(siteId)) {
-				List<String> specialNeeds = member.getAdditionalNotes();
+				row.add(member.getStudentNumber());
+			}
+
+			if (this.sakaiProxy.getViewCandidateDetails(siteId)) {
+				List<String> specialNeeds = member.getSpecialNeeds();
```
Comment (bbbbgarcia) [inline roster2/tool/src/java/org/sakaiproject/roster/tool/entityprovider/RosterPOIEntityProvider.java:492]: Couldn't all these 3 conditions go under the same if?

File: roster2/tool/src/java/org/sakaiproject/roster/tool/entityprovider/RosterPOIEntityProvider.java [hunk 2]
```diff
@@ -646,20 +646,20 @@ private List<String> createColumnHeader(final String viewType, final String site
...
-			header.add(rl.getString("facet_specialNeeds"));
+			header.add(rl.getString("facet_additionalNotes"));
 		}
 
 		if (this.sakaiProxy.getViewCandidateDetails(siteId)) {
-			header.add(rl.getString("facet_additionalNotes"));
+			header.add(rl.getString("facet_studentNumber"));
+		}
+
+		if (this.sakaiProxy.getViewCandidateDetails(siteId)) {
```
Comment (bbbbgarcia) [inline roster2/tool/src/java/org/sakaiproject/roster/tool/entityprovider/RosterPOIEntityProvider.java]: Couldn't all these 3 conditions go under the same if?

**Review by bbbbgarcia — CHANGES_REQUESTED** [review]

File: roster2/tool/src/java/org/sakaiproject/roster/tool/entityprovider/RosterPOIEntityProvider.java (hunk 1 above)
Comment (stetsche) [reply roster2/tool/src/java/org/sakaiproject/roster/tool/entityprovider/RosterPOIEntityProvider.java:492]: Yes!

Comment (stetsche) [reply roster2/tool/src/java/org/sakaiproject/roster/tool/entityprovider/RosterPOIEntityProvider.java:496]: Yes!

File: roster2/tool/src/java/org/sakaiproject/roster/tool/entityprovider/RosterPOIEntityProvider.java (hunk 2 above)
Comment (stetsche) [reply roster2/tool/src/java/org/sakaiproject/roster/tool/entityprovider/RosterPOIEntityProvider.java]: Yes!

**Comment by stetsche** [comment]
@bgarciaentornos Thanks, I applied the changes

File: roster2/bundle/src/bundle/roster_es.properties [hunk 3]
```diff
@@ -63,8 +63,8 @@ profile_email=Correo
 profile_picture_alt=Foto de
 
 # Additional info
-additionalInfo_button_text=Ver informaci\u00f3n adicional
-additionalInfo_modal_title=Informaci\u00f3n adicional del usuario {0}
+additionalInfo_button_text=Ver medidas de apoyo a la doc\u00e8ncia
+additionalInfo_modal_title=Medidas de apoyo a la doc\u00e8ncia del usuario {0}
```
Comment (bbbbgarcia) [inline roster2/bundle/src/bundle/roster_es.properties]: Docencia doesn't have accent mark in Spanish (these 2 + 1)

**Comment by bbbbgarcia** [comment]
Thanks! Just saw one little thing in the Spanish wording, sorry I didn't mention it before...

**Comment by stetsche** [comment]
Fixed it :+1:

**Review by bbbbgarcia — APPROVED** [review]
Thanks, LGTM

**Pull request merged**